packages/@lwc/integration-karma/test/api/CustomElementConstructor-slotting/index.spec.js

```diff
@@ -0,0 +1,15 @@
+import Test from 'x/test';
+
+describe('Custom Element slotting', () => {
+    const tagName = 'x-custom-element-slotting'; // unique tagName per file because we can't unregister custom elements
+    if (process.env.NATIVE_SHADOW) {
+        it('should render slotted content at the right place', () => {
+            const elm = document.createElement(tagName);
+            elm.attachShadow({ mode: 'open' });
+            elm.shadowRoot.innerHTML = 'before<slot></slot>after';
+            elm.innerText = 'Slotted';
+            document.body.appendChild(elm);
+            customElements.define(tagName, Test.CustomElementConstructor);
```

Comment:
Worth adding a comment pointing out that we're deliberately calling `define` _after_ the element is already in the DOM, to test upgrading an existing element.

Reply:
This comment is not needed now because it's part of the suite that tests implicit hydration and the test description clearly mentions that it occurs if customElements.define is called after the element is defined.